src/sandstorm/web-session.capnp

```diff
@@ -111,6 +147,48 @@ interface WebSession @0xa50711a14d35a8ce extends(Grain.UiSession) {
     encoding @2 :Text;  # Content-Encoding header (optional).
   }
 
+  struct PropfindContent {
```

Comment:
Since all these `Content` structs turn out to be the same, I think we should probably define a common `Content` struct instead. For now, use it just for the new methods; don't remove `PostContent` nor `PutContent`. Add a comment:

    // TODO(apibump): Remove PostContent and PutContent, replacing them with Content.

Then when we do an API revision we can fix this problem.